server/routes.json

```diff
@@ -43,5 +43,10 @@
         "pattern": "/wedo",
         "view": "wedo2",
         "title": "LEGO WeDo 2.0"
+    },
+    {
```

Comment:
Sorry to switch things up on you – it just so happens we merged in some changes to the way routes work, so it should read:
```(json)
{
    "name": "guidelines",
    "pattern": "^/community_guidelines$",
    "view": "guidelines/guidelines",
    "title": "Scratch Community Guidelines"
}
```